app/models/expense.rb

```diff
@@ -7,4 +7,8 @@ class Expense < ActiveRecord::Base
 
   has_attached_file :document
   validates_attachment_content_type :document, content_type: ['application/pdf', 'image/jpeg', 'image/png']
+
+  scope :joinuser, (lambda do |param|
+    joins(:user).where("email like '%#{param}%'")
+  end)
 end
```

Comment:
I had exactly your line but i changed it because Travis. Only this way i have no warnings on Travis.